### PR TITLE
My First Pull Request

### PR DESCRIPTION
**Pull request by braincee**
1. Added setup for linters
2.  Added and updated the main index.html. 
3. Added and updated the main style.css.
4.  Added and updated the gitignore files.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -348,3 +348,8 @@ MigrationBackup/
 
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
+
+#test
+test.md
+# .gitignore
+node_modules
```
Comment (mateo951) [inline .gitignore:355]: Great job! You ignored the necessary directory and file ✔

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @braincee,

Welcome to Microverse and congrats on making your first project 💪😎 

📝👌 **Your project highlights** 🔥😎 
- Nice README file 🚀 Well done!
- Great use of GitHub Flow 🤯
- Linters are working as expected 🙌
- Commit messages 🔥😎🔥

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Comment by mateo951** [comment]
Please add a short summary describing the introduced changes in more detail.

For example, you could go with something like this🤓
  - Added setup for linters
  - Added gitignore file with ignored files/directories 
  - Added main structure (index.html / style.css)

**Comment by braincee** [comment]
> Please add a short summary describing the introduced changes in more detail.
> 
> image
> 
> For example, you could go with something like this🤓
> 
> * Added setup for linters
> * Added gitignore file with ignored files/directories
> * Added main structure (index.html / style.css)

**Comment by braincee** [comment]
I have added all the short descriptive summaries in detail.

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @braincee , 

Great work on making the changes requested by the previous reviewer 👏 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by braincee** [comment]
Thanks @micheaol 
472108170_THANK_YOU_STICKER_400px

**Pull request merged**